glean-core/preview/src/lib.rs

```diff
@@ -22,12 +22,12 @@
 //! let cfg = Configuration {
 //!     data_path: "/tmp/data".into(),
 //!     application_id: "org.mozilla.glean_core.example".into(),
-//!     upload_enabled: true,
 //!     max_events: None,
 //!     delay_ping_lifetime_io: false,
 //!     channel: None,
 //! };
 //! glean_preview::initialize(cfg, ClientInfoMetrics::unknown())?;
+//! glean_preview::set_upload_enabled(true);
```

Comment:
I think that `glen_preview::initialize`, in this case, should take care of calling the right thing internally.

Reply:
Good point.